### PR TITLE
7. Post page impl

### PR DESCRIPTION
**Pull request by petroborovets**
Post page content implementation, few new components:
Screenshot 2024-05-11 at 15 47 59

### CODE REVIEW

File: src/components/common/featuredSelection/featuredSectionEntry/FeaturedSectionEntry.jsx [hunk 1]
```diff
@@ -7,7 +7,7 @@ import {Link} from "react-router-dom";
 /* eslint-disable react/prop-types */
 function FeaturedSectionEntry(props) {
     const {id, image, category, title, date} = props
-    const categoryName = categoriesJson?.categories?.filter(c => c.id === category)[0]?.name || 'Unknown Category';
```
Comment (petroborovets) [inline src/components/common/featuredSelection/featuredSectionEntry/FeaturedSectionEntry.jsx:10]: Hm, this one I will revert, it should be in place.

File: src/components/common/post/section/child/PostSectionChild.jsx [hunk 2]
```diff
@@ -0,0 +1,30 @@
+import s from "./PostSectionChild.module.css";
+import {ORIGIN_URL} from "../../../../../constants/pathConstants.js";
+
+/* eslint-disable react/prop-types */
+function PostSectionChild(props) {
+    const sectionChild = props.sectionChild;
+
+    let element;
+    if (sectionChild.type === 'image') {
```
Comment (petroborovets) [inline src/components/common/post/section/child/PostSectionChild.jsx:9]: @boandriy , can we improve this code somehow?

File: src/components/common/post/section/PostSection.jsx [hunk 3]
```diff
@@ -0,0 +1,20 @@
+import s from './PostSection.module.css';
+import PostSectionChild from "./child/PostSectionChild.jsx";
+
+/* eslint-disable react/prop-types */
+function PostSection(props) {
+    const section = props.section;
```
Comment (boandriy) [inline src/components/common/post/section/PostSection.jsx:6]: `const { section } = props;`

File: src/components/header/Header.jsx [hunk 4]
```diff
@@ -23,12 +19,19 @@ function Header() {
                         <Link to="/pebo/about" title="">About</Link>
                     </li>
 
-                    <li className={s.header_nav_li}>
-                        <Link to="/pebo/contact" title="">Contact</Link>
-                    </li>
+                    {/*<li className={s.header_nav_li}>*/}
```
Comment (petroborovets) [inline src/components/header/Header.jsx:22]: I moved contact to far right in navbar, and hidden the search that did not work

File: src/components/common/post/section/PostSection.jsx (hunk 3 above)
Comment (petroborovets) [reply src/components/common/post/section/PostSection.jsx]: Fixed.

File: src/components/common/post/section/child/PostSectionChild.jsx (hunk 2 above)
Comment (boandriy) [reply src/components/common/post/section/child/PostSectionChild.jsx:9]: Need a call hare.

File: src/components/common/post/section/child/PostSectionChild.jsx [hunk 5]
```diff
@@ -0,0 +1,30 @@
+import s from "./PostSectionChild.module.css";
+import {ORIGIN_URL} from "../../../../../constants/pathConstants.js";
+
+/* eslint-disable react/prop-types */
+function PostSectionChild(props) {
+    const sectionChild = props.sectionChild;
```
Comment (boandriy) [inline src/components/common/post/section/child/PostSectionChild.jsx:6]: const { sectionChild } = props;

**Review by boandriy — APPROVED** [review]

File: src/components/common/post/section/PostSection.jsx [hunk 6]
```diff
@@ -0,0 +1,20 @@
+import s from './PostSection.module.css';
+import PostSectionChild from "./child/PostSectionChild.jsx";
+
+/* eslint-disable react/prop-types */
+function PostSection(props) {
```
Comment (boandriy) [inline src/components/common/post/section/PostSection.jsx:5]: Just FYI you can also do it like this - `function PostSection({ section }) {` - deconstucting props

**Review by boandriy — APPROVED** [review]

**Pull request merged**